members/federal_us_firstlady_jill_biden.yaml

```diff
@@ -16,7 +16,7 @@ contact_form:
       - name: Your Prefix
         selector: select[id='Contact.Salutation']
         value: $NAME_PREFIX
-        required: true
+        required: False
```

Comment:
not 100% sure if this matters, but can you use lower case `false` instead?

Reply:
fixed. I also changed `True` to lowercase